azure-pipelines-integration-dartlab.yml

```diff
@@ -54,6 +62,7 @@ stages:
         filePath: $(DartLab.Path)\Scripts\VisualStudio\Build\Get-VisualStudioDropName.ps1
         arguments: -DropNamePrefix 'Products' -VstsDropUrlsJson '$(Pipeline.Workspace)\VisualStudioBuildUnderTest\BuildArtifacts\VstsDropUrls.json' -OutVariableName 'VisualStudio.BuildUnderTest.ProductsDropName'
     deployAndRunTestsStepList:
+    - checkout: RoslynMirror
```

Comment:
Since we do not specify `- checkout: self` we are pulling the source from our mirror instead of GH.

Reply:
Does this run the full `git` checkout or the modified one that we built which does a minimal checkout?

Reply:
This is AzDO running its full `git` checkout.  I will have to see how extra git resources are managed to determine if we can update our checkout task.

Reply:
I don't think it's anything that should block this change but is something that we should keep an eye on. The frequency of our checkouts in normal CI made this a problem, doubt this hits as frequently hence less of a trigger point.